install/readme.md

```diff
@@ -0,0 +1,80 @@
+# SODA Experience Installer
+
+The SODA Experience installer enables users to install the SODA Authentication module, Keystone, and the SODA Dashboard independently.  
+
+
+## Pre-requisites
+
+- This installer is currently tested and works on **Ubuntu 20.04**. 
+
+- Python  >= 3.8.10 ***(This installer may not work on Ubuntu 18.04 since the python3 version may not be supported.)***
+
+- Install Docker version > 20.10.21
+
+- Export the IP address of the host machine to the environment variable
+
+```sh
+export HOST_IP=<your_host_ip> # Replace with your actual host IP
+echo $HOST_IP
+```
+
+- Export the desired SODA Release version to the environment variable
+
+```sh
+export SODA_RELEASE_VERSION=v1.7.0 # Replace with SODA release version or docker tag for dashboard.
```

Comment:
Rename SODA_RELEASE_VERSION to DASHBOARD_RELEASE_VERSION

Reply:
Done